nvidia_nims/vista_3d_remote_nim.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>Do you think adding two screenshots here would improve the overall clarity and visual appeal?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Added.

Reply:
You may need using `![Figure](Figure.png)`, the screenshots not shown as expected here: https://github.com/Project-MONAI/tutorials/blob/95eff84dfbadcfba066c3b09e88176d905e1aff7/nvidia_nims/vista_3d_remote_nim.ipynb